osu.Game.Rulesets.Taiko/UI/DrawableTaikoRuleset.cs

```diff
@@ -70,7 +71,7 @@ protected override void Update()
         protected virtual double ComputeTimeRange()
         {
             // Taiko scrolls at a constant 100px per 1000ms. More notes become visible as the playfield is lengthened.
-            const float scroll_rate = 10;
+            const float scroll_rate = 10 / TaikoBeatmapConverter.VELOCITY_MULTIPLIER;
```

Comment:
I _think_ I see what this is doing but I'd say it is rather opaque... Especially so that this division kinda ruins the preceding inline comment.

What would you think about moving this somewhere else with some additional commentary? e.g.

```diff
diff --git a/osu.Game.Rulesets.Taiko/UI/DrawableTaikoRuleset.cs b/osu.Game.Rulesets.Taiko/UI/DrawableTaikoRuleset.cs
index 88085dfe97..8702f1ef38 100644
--- a/osu.Game.Rulesets.Taiko/UI/DrawableTaikoRuleset.cs
+++ b/osu.Game.Rulesets.Taiko/UI/DrawableTaikoRuleset.cs
@@ -71,14 +71,18 @@ protected override void Update()
         protected virtual double ComputeTimeRange()
         {
             // Taiko scrolls at a constant 100px per 1000ms. More notes become visible as the playfield is lengthened.
-            const float scroll_rate = 10 / TaikoBeatmapConverter.VELOCITY_MULTIPLIER;
+            const float scroll_rate = 10;
 
             // Since the time range will depend on a positional value, it is referenced to the x480 pixel space.
             // Width is used because it defines how many notes fit on the playfield.
             // We clamp the ratio to the maximum aspect ratio to keep scroll speed consistent on widths lower than the default.
             float ratio = Math.Max(DrawSize.X / 768f, TaikoPlayfieldAdjustmentContainer.MAXIMUM_ASPECT);
 
-            return (Playfield.HitObjectContainer.DrawWidth / ratio) * scroll_rate;
+            // Stable internally increased the slider velocity of objects by a factor of `VELOCITY_MULTIPLIER`.
+            // To simulate this, we shrink the time range by that factor here.
+            // This, when combined with the rest of the scrolling ruleset machinery (see `MultiplierControlPoint` et al.),
+            // has the effect of increasing each multiplier control point's multiplier by `VELOCITY_MULTIPLIER`, ensuring parity with stable.
+            return (Playfield.HitObjectContainer.DrawWidth / ratio) * scroll_rate / TaikoBeatmapConverter.VELOCITY_MULTIPLIER;
         }
 
         protected override void UpdateAfterChildren()
```